src/routes/article-list.ts

```diff
@@ -2,8 +2,9 @@ import Router from '@koa/router';
 import { Context, Middleware, Next } from 'koa';
 import { hydra, rdf, schema } from 'rdf-namespaces';
 import Routes from './index';
+import { Articles } from '../adaptors/articles';
```

Comment:
The interface shouldn't be under `adaptors`.

Reply:
I'm moving it to the `/src` folder or should we organise it somewhere else?